sklearn/utils/tests/test_extmath.py

```diff
@@ -457,6 +458,16 @@ def test_incremental_variance_ddof():
                                 calculated_variances, 6)
             assert_equal(incremental_count, sample_count)
 
+
+def test_eigen_sign_flip():
+    # Testing that sign flip is working & largest value has positive sign
+    random_state = np.random.RandomState(36)
+    data = random_state.randn(5, 5)
```

Comment:
combine both lines to get
`data = np.random.RandomState(36).randn(5, 5)`

or if you feel like, could you instead refactor all these out ( since they are repeated across multiple tests ) to the top like
```
rng = np.random.RandomState(0)
```
and use `rng` everywhere inside the tests instead? ( preferably in a separate commit )...

like `rng.randn(5, 5)` etc...